src/main/java/seedu/lifeasier/ui/Ui.java

```diff
@@ -582,4 +582,20 @@ public void showInvalidRecurrencesError() {
         System.out.println(colourTextRed("Recurrences must be a positive integer!"));
         printSeparator();
     }
+
+    public void showEnterDisplayKeywordMessage() {
+        System.out.println(colourTextCyan("To see your schedule, please enter: ") + "week/today/tomorrow");
+        printSeparator();
+    }
+
+    public void showInvalidDisplayKeyword() {
+        System.out.println("Invalid display parameter! Try 'week', 'today' or 'tomorrow' instead!");
+        printSeparator();
+    }
```

Comment:
I think that since it is an error, you could consider naming it as such, and printing it in red.
```suggestion
    public void showInvalidDisplayKeywordError() {
        System.out.println(colourTextRed("Invalid display parameter! Try 'week', 'today' or 'tomorrow' instead!"));
        printSeparator();
    }```